demo/api/src/app.module.ts

```diff
@@ -143,6 +145,13 @@ export class AppModule {
                     directory: `${config.blob.storageDirectoryPrefix}-public-uploads`,
                     acceptedMimeTypes: ["application/pdf", "application/x-zip-compressed", "application/zip"],
                 }),
+                ...(config.contentGeneration.apiKey && config.contentGeneration.apiUrl && config.contentGeneration.deploymentId
```

Comment:
I'd move this check into config.ts and make `config.contentGeneration` optional as a whole

Reply:
Done a5ced656e00f442d1e66e6f2cab70e03f100e462